packages/plugins/automatic-persisted-queries/src/hash-sha256.ts

```diff
@@ -0,0 +1,74 @@
+const webCrypto: Crypto | undefined = globalThis.crypto
```

Comment:
We can use this instead;
https://github.com/ardatan/whatwg-node/blob/master/packages/fetch/dist/create-node-ponyfill.js#L98